kitt_bar_app/plugin.rb

```diff
@@ -44,7 +44,7 @@ def header
 	end
 
```

Comment:
here we wanted to get all plugin headers of all current ticket and we were getting Ticket instances instead because of the wrongly named method in current_batch.rb